index.bs

```diff
@@ -127,57 +136,47 @@ companies with large portfolios of disparate products all under the same domain.
 * Similarly, provide a way for content authors to split their applications
   into logical modules with origin level separation without using different real
   origins. Content authors should not have to choose between putting all of
-  their content in the same origin, on different real origins, or putting
+  their content in the same origin, on different physical origins, or putting
   content in anonymous unique origins (sandboxes).
 
 * Provide safe defaults but also make it as simple as possible to retrofit
-  legacy applications on the same real origin with minimal programmatic changes.
+  legacy applications on the same physical origin with minimal programmatic changes.
   This includes providing security-model opt-outs where necessary.
 
-Issue: Not sure how to actually refer to 'real origins'. This is a terrible
-name, and we need a better way to talk about them. Maybe physical origin?
-Traditional origin? (jww)
-
 ## Use Cases/Examples ## {#usecases}
 
 We see effectively three different use cases for Per-page Suborigins:
 
-1. Separating distinct applications that happen to be served from the same
+1. Separating distinct applications served, due to deployment issues, from the same
```

Comment:
A little awkward. Maybe rephrase as "Separating distinct applications that are served from the same domain because of deployment issues, but do not need..."?